pkg/dependencies/min_version.go

```diff
@@ -15,6 +14,8 @@ import (
 	"github.com/ava-labs/avalanche-cli/pkg/constants"
 )
 
+var UpdateSubnetEVMInstruction = "To update the blockchain's Subnet-EVM version, call avalanche blockchain upgrade vm <blockchainName> --config --version <version>"
```

Comment:
please don't use blockchain upgrade vm, but blockchain configure. it is planned to move everything, upgrades and vm binary/source changes, into blockchain configure